src/database/sql/sequelize/database.connector.sequelize.ts

```diff
@@ -36,6 +36,7 @@ export class DatabaseConnector_Sequelize implements IDatabaseConnector {
             };
 
             const sequelize = new Sequelize(config.database, config.username, config.password, options);
+            sequelize.addModels([__dirname + '/models/biometrics'])
```

Comment:
Please add this to the modelsPath array above on line no. 24 and remove from here...
e.g.
```
const modelsPath = [
  __dirname + '/models',
  __dirname + '/models/biometrics'
  ];
```